enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1298,6 +1299,43 @@ func (p *PebbleCache) lookupFileMetadata(ctx context.Context, iter pebble.Iterat
 	return md, err
 }
 
+func (p *PebbleCache) lookupLastAccessTime(iter pebble.Iterator, key filestore.PebbleKey) (int64, error) {
+	for version := p.maxDatabaseVersion(); version >= p.minDatabaseVersion(); version-- {
+		keyBytes, err := key.Bytes(version)
+		if err != nil {
+			return 0, err
+		}
+		if iter.SeekGE(keyBytes) && bytes.Equal(iter.Key(), keyBytes) {
+			return getLastAccessUsec(iter.Value()), nil
+		}
+	}
+	return 0, status.NotFoundErrorf("key %q not found", key)
+}
+
+func getLastAccessUsec(b []byte) int64 {
```

Comment:
nit: comment why this is this way and link to the proto tag field definition (or explain that lastAccessUsecTag matches it)